styx-local-files-source/src/main/java/com/spotify/styx/Yaml.java

```diff
@@ -35,7 +36,8 @@
       .setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE)
       .enable(ACCEPT_SINGLE_VALUE_AS_ARRAY)
       .registerModule(new Jdk8Module())
-      .registerModule(new JavaTimeModule());
+      .registerModule(new JavaTimeModule())
+      .registerModule(new AutoMatterModule());
```

Comment:
Just as a reminder, this shall be applied to internal ScheduleSource as well. Maybe open an internal PR already now.